bin/api_package_docker

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+###########################################################################
+# Creates Docker image and uploads it to DockerHub https://hub.docker.com/r/mysteriumnetwork/mysterium-api/
+#
+# To create a release, simply execute:
+#> bin/api_package_docker version [docker_tag]
+
+# Usage example:
+#> bin/api_package_docker 0.0.1
+#> bin/api_package_docker 0.0.1 latest
+
+VERSION=$1
+if [ -z "$VERSION" ]; then
+    printf "\e[0;31m%s\e[0m\n" "Missing version!"
+    exit 1
+fi
+
+DOCKER_IMAGE="mysteriumnetwork/mysterium-api"
+DOCKER_TAG=$2
+if [ -z "$DOCKER_TAG" ]; then
+    DOCKER_TAG=$VERSION
+fi
+
+printf "Building Docker image '$DOCKER_IMAGE:$DOCKER_TAG' ..\n" \
+    && docker build \
+        --file Dockerfile \
+        --tag $DOCKER_IMAGE:$DOCKER_TAG \
+        . \
+    && docker push $DOCKER_IMAGE:$DOCKER_TAG
```

Comment:
How will we deploy updates to api and dashboard?
Could you update the instructions in https://mysteriumnetwork.atlassian.net/wiki/spaces/MN/pages/27525121/Discovery+API+Deployment ?
Also, since only limited number of people have permissions to push to DockerHub, does that mean that I won't be able to deploy updates?

Reply:
Fixed